src/actions/index.js

```diff
@@ -316,9 +324,12 @@ export function notifyObservers(payload) {
              * of a controller change.
              * for example, perhaps the user has hidden one of the observers
              */
+
+            const {paths} = getState();
+
             if (
                 controllersInFutureQueue.length === 0 &&
```

Comment:
I don't think `controllersInFutureQueue` will be correct here, since the `controllers` variable set above:
```
            const controllers = InputGraph.hasNode(outputIdAndProp)
                 ? InputGraph.dependantsOf(outputIdAndProp)
                 : [];
```
Would be `[]` whenever we are doing multiple outputs.

Reply:
The controllers variable has the required inputs.

Reply:
Oh I didn't know those comments were posting :smile:  Makes sense after reading below